specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+          "type": "string",
+          "enum": [
+            "Invalid",
+            "AlreadyExists"
+          ],
+          "x-ms-enum": {
+            "name": "Reason",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "Invalid"
+              },
+              {
+                "value": "AlreadyExists"
+              }
+            ]
+          },
+          "readOnly": true
+        },
+        "message": {
+          "description": "Required if nameAvailable == false. Localized. If reason == invalid, provide the user with the reason why the given name is invalid, and provide the resource naming requirements so that the user can select a valid name. If reason == AlreadyExists, explain that is already in use, and direct them to select a different name.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "TenantBackfillStatusResult": {
+      "description": "The tenant backfill status",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the management group. For example, 00000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "status": {
+          "description": "The status of the Tenant Backfill",
+          "type": "string",
+          "enum": [
+            "NotStarted",
+            "NotStartedButGroupsExist",
+            "Started",
+            "Failed",
+            "Cancelled",
+            "Completed"
+          ],
+          "x-ms-enum": {
+            "name": "Status",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "NotStarted"
+              },
+              {
+                "value": "NotStartedButGroupsExist"
+              },
+              {
+                "value": "Started"
+              },
+              {
+                "value": "Failed"
+              },
+              {
+                "value": "Cancelled"
+              },
+              {
+                "value": "Completed"
+              }
+            ]
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "ManagementGroupListResult": {
+      "description": "Describes the result of the request to list management groups.",
+      "properties": {
+        "value": {
+          "description": "The list of management groups.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupInfo"
+          }
+        },
+        "@nextLink": {
+          "description": "The URL to use for getting the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ManagementGroupInfo": {
```

Comment:
>ManagementGroupInfo [](http://example.com/codeflow?start=5&length=19)

If you snap to the RPC re: collection GET request and 202 responses, you won't need this definition (or the ManagementGroupInfoProperties definition) any more.